Forge.TreeWalker/contracts/ForgeTree.cs

```diff
@@ -192,6 +192,14 @@ public class RetryPolicy
         /// </summary>
         [DataMember]
         public long MaxBackoffMs { get; private set; }
+
+        /// <summary>
+        /// This setting is specifically for RetryPolicyType.FixedCount.
+        /// Max retry is the maximum number of times to attempt to run an action before failing. 
+        /// The default value is 0; if value is not specified no retries will run. 
```

Comment:
>The default value is 0; if value is not specified no retries will run. [](start = 12, length = 70)

Hmm.. Should we set the default value to 1 instead of 0? That will make sure we execute once. Accidentally leaving this as 0 would be confusing for the end-user when their action doesn't execute at all. What do you think?